tests/admin_changelist/tests.py

```diff
@@ -1741,6 +1741,44 @@ def test_modifier_allows_multiple_section(self):
             self.assertIs(c.get_property("checked"), True)
         self.assertIs(checkboxes[-1].get_property("checked"), False)
 
+    def test_selection_counter_is_synced_when_page_is_shown(self):
+        from selenium.webdriver.common.by import By
+
+        self.admin_login(username="super", password="secret")
+        self.selenium.get(self.live_server_url + reverse("admin:auth_user_changelist"))
+
+        form_id = "#changelist-form"
+        first_row_selector = "%s #result_list tbody tr:first-child" % form_id
```

Comment:
I would use [f-strings](https://docs.python.org/3/tutorial/inputoutput.html#formatted-string-literals), I think is more clear